src/react-components/room/useResizeViewport.js

```diff
@@ -73,7 +73,7 @@ export function useResizeViewport(viewportRef, store, scene) {
           return;
         }
 
-        const canvasRect = entries[0].contentRect;
+        const canvasRect = { width: window.innerWidth, height: window.innerHeight };
```

Comment:
@netpro2k @keianhzo Do you think this is a proper way to set window-full-size canvas?

Reply:
This looks good to me, I think it should work well but just curious, why did you change `entries[0].contentRect`?

Reply:
Oh ok, thanks for the explanation. 

The `useResizeViewport` hook takes a node and resizes the viewport according to the latest node size. With this change we are always setting the viewport to the window size which works in this case but breaks the hook contract as we are not resizing based on the `viewportRef` anymore. It works in this case but it might be confusing for people using the hook in the future so I'd rather keeping the hook contract as it was and  pass the right viewport reference to it.

Once the `useResizeViewport`code has been reverted, you can assign the `viewportRef` to the room layout div in `RoomLayout.js`. That div should cover the entire window if I'm right.
```javascript
return (
    <div
      className={classNames(styles.roomLayout, { [styles.objectFocused]: objectFocused }, className)}
      ref={viewportRef}
      {...rest}
    >
      {sidebar && <div className={classNames(styles.sidebar, sidebarClassName)}>{sidebar}</div>}
      <div className={classNames(styles.modalContainer, styles.viewport)}>{modal}</div>
      {(toolbarLeft || toolbarCenter || toolbarRight) && (
        <Toolbar
          className={classNames(styles.main, styles.toolbar, toolbarClassName)}
          left={toolbarLeft}
          center={toolbarCenter}
          right={toolbarRight}
        />
      )}
      <div className={classNames(styles.main, styles.viewport, { [styles.streaming]: streaming }, viewportClassName)}>
        {viewport}
      </div>
    </div>
  );
```

Sorry for missing this on my first review.

Reply:
Thanks, updated.

Reply:
You also need to remove the `viewportRef` from the last div otherwise otherwise we can get unexpected behavior.

Reply:
Updated, thanks.